db/statistics.go

```diff
@@ -1260,10 +1260,118 @@ func GetValidatorIncomeHistory(validatorIndices []uint64, lowerBoundDay uint64,
 
 func WriteChartSeriesForDay(day int64) error {
 	startTs := time.Now()
+	err := WriteExecutionChartSeriesForDay(day)
```

Comment:
Would it make sense to start a go routine for this? (and WriteConsensusChartSeriesForDay)

Reply:
good idea, fixed